comms/EclCC.cpp

```diff
@@ -49,6 +47,7 @@ class CEclCC : public IEclCC, public clib::CLockableUnknown
 public:
     std::_tstring m_errors;
     std::_tstring m_warnings;
+    mutable CEclMeta m_eclMeta;
```

Comment:
Probably doesn't need to be mutable?

Reply:
@GordonSmith it turns out that it doesn't need to be mutable if GetAutoC is changed to a non-constant. Going that route.